src/views/templates/details/hooks/useIsTemplateEditable.ts

```diff
@@ -0,0 +1,32 @@
+import { TemplateModel, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useAccessReview } from '@openshift-console/dynamic-plugin-sdk';
+
+import { isCommonVMTemplate } from '../../utils/utils';
+
+/**
+ * This hook checks the user capabilities of editing templates in a namespace.
+ * Checks user permissions and template characteristics.
+ * The hook does not check if the user has permissions for that particular template
+ * as this behavior would compromise performarce in pages were a lot of templates are involved like template list.
+ * @param template template to check
+ * @returns boolean value
+ */
+const useIsTemplateEditable = (template: V1Template) => {
+  const [canUpdateTemplate, canUpdateLoaded] = useAccessReview({
```

Comment:
the `useAccessReview`  has the loading meaning for the second element in the returning array unlike the `useK8sWatchResource` which has the meaning of loaded, please refer to the hook in console https://github.com/openshift/console/blob/97368e5be5b2b2e92cdee35e42b2acaf8c6471f6/frontend/public/components/utils/rbac.tsx#L90-L125
you can also see the comments in the rbac.d.ts file in your local env

please change canUpdateLoaded to canUpdateLoading

Reply:
thanks man